app/templates/emailModal.html

```diff
@@ -0,0 +1,66 @@
+<script src="/static/js/event_list.js"></script>
+<div class="modal fade" id="emailModal" tabindex="-1" aria-labelledby="emailModalLabel" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-scrollable modal-lg">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title m-auto" id="emailModalLabel">'Event/Program' Email Template</h4>
+        <button type="button" class="btn-close position-absolute end-0 me-3" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <h6 class="text-center">Last Email was sent to ____ on 10/20/20</h6>

```

Comment:
We should either remove this, implement it, or indicate that it is a nonfunctioning placeholder